src/client/wieldmesh.cpp

```diff
@@ -481,9 +485,9 @@ void WieldMeshSceneNode::setNodeLightColor(video::SColor color)
 			video::SMaterial &material = m_meshnode->getMaterial(i);
 			material.EmissiveColor = color;
 		}
-	} else {
-		setColor(color);
 	}
+
+	setColor(color);
```

Comment:
I made this unconditional because otherwise something overwrites colors, or mesh is created asynchronously in separate thread and at the time `WieldMeshSceneNode` constructor finishes, buffers are not there yet. Besides, it was working like this before #9537